HomeWork-6/src/main/java/com/example/homework6/repository/TaskRepositoryImpl.java

```diff
@@ -0,0 +1,140 @@
+package com.example.homework6.repository;
+
+import com.example.homework6.enums.TaskPriority;
+import com.example.homework6.enums.TaskStatus;
+import com.example.homework6.model.Task;
+import com.example.homework6.model.User;
+import com.example.homework6.repository.dao.TaskRepository;
+import com.example.homework6.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class TaskRepositoryImpl implements TaskRepository {
+
+    private final Connection connection;
+    private final UserService userService;
```

Comment:
It,s a mistake. The repository shouldn't know anything about services

Reply:
UserService was changed to UserRepository in TaskRepositoryImpl. In the future, I may change the implementation